tests/ui/impl-trait/in-assoc-type-unconstrained.stderr

```diff
@@ -36,7 +36,7 @@ LL |         fn method() -> Self::Ty;
    |                        ^^^^^^^^
    = note: expected signature `fn() -> <() as compare_method::Trait>::Ty`
               found signature `fn() -> ()`
-note: this item must have the opaque type in its signature in order to be able to register hidden types
+note: this item must have a `#[define_opaques(<() as compare_method::Trait>::Ty)]` attribute to be able to define hidden types
```

Comment:
This is not possible, is it? Please change the error for ATPIT or probably just add a `FIXME` for now